docs/user/contextplugin.md

```diff
@@ -0,0 +1,19 @@
+# FlowForge Persistent Context Plugin
+
+Projects running within FlowForge include a plugin that make it possible
```

Comment:
What about:

>>>
When developing flows there's might be a need to share data between flow executions.
This data is ephemeral by default. When your FlowForge is a Premium FlowForge, there's
persistent context available. This surfaces stack upgrades, among other scenarios.
>>>